css/mail.css

```diff
@@ -602,6 +602,9 @@
 .icon-sent {
 	background-image: url('../img/sent.svg');
 }
+.icon-archive {
+	background-image: url('../img/archive.svg');
```

Comment:
@biolumin This merge is icon-related and adds the archive icon here (does this answer your question?)